firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -176,7 +177,28 @@ class BrowserCoordinator: BaseCoordinator,
                                                    modalStyle: .overFullScreen)
         let sheet = PhotonActionSheet(viewModel: viewModel, windowUUID: windowUUID)
         sheet.modalTransitionStyle = .crossDissolve
-        present(sheet, animated: true)
+        present(sheet)
+    }
+
+    func showEditBookmark(parentFolder: FxBookmarkNode, bookmark: FxBookmarkNode) {
+        guard !childCoordinators.contains(where: { $0 is BookmarksCoordinator }) else { return }
```

Comment:
I think we don't need this line. We use it for example in the case of the Settings coordinator since the settings coordinator can be opened, and the user opens the application through a deeplink. The deeplink can then be handled by the already existing child coordinator.

In the present case, we don't handle deeplinks flow within that page + the bookmarksCoordinator is supposed to be always removed whenever the flow is done. 

Further note as well, in your case you have `{ $0 is BookmarksCoordinator }`. I don't think we have a case where the BookmarksCoordinator would be a child of itself. 

With all that said, I think we can remove this line. What do you think?
```suggestion
```

Reply:
Mostly just copied it for consistency with how we create other coordinators, and it made sense, but you're right, I don't see how it would be possible to get to this codepath twice without dismissing the original coordinator, so let's get rid of it!